src/plotman/resources/plotman.yaml

```diff
@@ -3,14 +3,17 @@
 # https://github.com/ericaltendorf/plotman/wiki/Configuration#versions
 version: [2]
 
-logging:
-        # One directory in which to store all plot job logs (the STDOUT/
-        # STDERR of all plot jobs).  In order to monitor progress, plotman
-        # reads these logs on a regular basis, so using a fast drive is
-        # recommended.
-        plots: /home/chia/chia/logs
-        # transfers:
-        # application:
+#logging:
+#        # One directory in which to store all plot job logs (the STDOUT/
+#        # STDERR of all plot jobs).  In order to monitor progress, plotman
+#        # reads these logs on a regular basis, so using a fast drive is
+#        # recommended.
+#        # For Linux, these paths default to a directory under ~/.local/share/plotman/
+#         plots: <directory>
+#         transfers: <directory>
+#        # For Linux, these paths default to a file at ~/.cache/plotman/log/
+#         application: <file>
+#         disk_spaces: <file>
```

Comment:
Intentional that this whole section is commented out?

Reply:
Yes.  It had one made up path that wouldn't be accurate for you unless you had a username of `chia`.  A couple people got caught up on that.  Leaving it not-present/commented-out uses defaults for all locations based on `appdirs`.